static_sites/labelling_librarian/README.md

```diff
@@ -0,0 +1,84 @@
+# Labelling Librarian
+
+A web-based tool that helps concept developers maintain the quality and consistency of our [labelled data](https://argilla.labs.climatepolicyradar.org/).
+
+The tool checks for issues with labelled datasets in the Argilla and generates a static website which:
+
+## Usage
+
+### Generating the static site
+
+To check for issues and generate a new version of the static site, run:
+
+```bash
+just generate-static-site labelling_librarian
+```
+
+The HTML files will be saved locally to the `static_sites/labelling_librarian/dist` directory.
+
+To preview the site locally, you can start a local server:
+
+```bash
+just serve-static-site labelling_librarian
+```
+
+The site will be available at `http://localhost:8080`.
+
+## Deployment
+
+The site is deployed using AWS CloudFront and S3.
+
+### Deploy the infrastructure
+
+This step is only required if you're making changes to the infrastructure itself - skip it if you're just updating the site content, and move straight on to [Pushing new data to s3](#pushing-new-data-to-s3).
```

Comment:
to remove the cost of change later on, is it worth combining these instructions with the one for the concept librarian?

Reply:
I think it's fine for the time being - IMO this is a case where being overly DRY would actually make it _harder_ to use these docs, rather than easier, given the differences with the vibe_check tool. Def worth revisiting to create a totally generic module when the number of these copy-pasted READMEs is higher though!

Reply:
ah cool, i didn't realise these instructions were different (i didn't check the others)